test-lib.sh

```diff
@@ -208,6 +208,40 @@ function ct_doc_content_old() {
 }
 
 
+# ct_mount_ca_file
+# ------------------
+# Check if /etc/pki/certs/RH-IT-Root-CA.crt file exists
+# return mount string for containers
+ct_mount_ca_file()
+{
+    local mount_parameter=""
+    if [ -n $NPM_REGISTRY ]; then
+        local ca_full_path="/etc/pki/ca-trust/source/anchors/RH-IT-Root-CA.crt"
+        if [ -f "$ca_full_path" ]; then
+            mount_parameter="-v $ca_full_path:$ca_full_path:ro,Z"
+        fi
+    fi
+    echo "$mount_parameter"
+}
+
+# ct_build_s2i_npm_variables URL_TO_NPM_JS_SERVER
+# ------------------------------
+# Function returns -e NPM_MIRROR and -v MOUNT_POINT_FOR_CAFILE
+function ct_build_s2i_npm_variables() {
+    local ca_full_path="/etc/pki/ca-trust/source/anchors/RH-IT-Root-CA.crt"
+    if [ ! -z $NPM_REGISTRY ]; then
```

Comment:
`-n` instead, to have the same check as in `ct_mount_ca_file`. Missing quotes here as well.

Reply:
Good point. `ct_mount_ca_file` is used by `ct_build_s2i_npm_variables` and `ct_npm_works` functions.

Reply:
I guess `ct_build_s2i_npm_variables` will be used only for `-e NPM_MIRROR` and `ct_mount_ca_file` will be used only for `-v <path>:<path>:`.

Reply:
I see nothing wrong with `ct_build_s2i_npm_variables` returing both `-e NPM_MIRROR` and `-v <path>:<path>:`